src/MessagingInput/MessagingInput.tsx

```diff
@@ -137,7 +137,7 @@ const MessagingInputRenderFunction: React.ForwardRefRenderFunction<MessagingInpu
             showUploadAttachmentButton={showUploadAttachmentButton}
             storeUpload={store}
             fileUploadCustomIcon={<MessagingAttachmentIcon />}
-            uploads={attachments && formUploadedAttachments(attachments)}
+            uploads={attachments.length > 0 && formUploadedAttachments(attachments)}
```

Comment:
Nit: Can we do an optional chain to handle the nil/undefined case? `attachments?.length > 0`

Reply:
nice! will do